.changeset/dirty-gorillas-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/ui-extensions': minor
+---
+
+Added end of support dates for older POS UI versions
```

Comment:
I don't think you need a changeset. Only need one if you're trying to get a new `unstable` release. This is just for docs

Reply:
^^